tensorflow_addons/optimizers/tests/conditional_gradient_test.py

```diff
@@ -571,409 +571,377 @@ def test_tensor_learning_rate_and_conditional_gradient_frobenius(dtype):
-                )
-                self.assertAllCloseAccordingToType(
-                    np.array(
-                        [
-                            1.0 * learning_rate
-                            - (1 - learning_rate) * lambda_ * 0.01 / norm1,
-                            1.0 * learning_rate
-                            - (1 - learning_rate) * lambda_ * 0.01 / norm1,
-                        ]
-                    ),
-                    self.evaluate(var1)[2],
-                )
-                # Step 2: the conditional_gradient contain the
-                # previous update.
-                if tf.executing_eagerly():
-                    cg_opt.apply_gradients(zip([grads0, grads1], [var0, var1]))
-                else:
-                    self.evaluate(cg_update)
-                # Check that the parameters have been updated.
-                self.assertAllClose(np.array([0, 0]), self.evaluate(var0)[0])
-                self.assertAllCloseAccordingToType(
-                    np.array(
-                        [
-                            (0 - (1 - learning_rate) * lambda_ * 0.1 / norm0)
-                            * learning_rate
-                            - (1 - learning_rate) * lambda_ * 0.1 / norm0,
-                            (0 - (1 - learning_rate) * lambda_ * 0.1 / norm0)
-                            * learning_rate
-                            - (1 - learning_rate) * lambda_ * 0.1 / norm0,
-                        ]
-                    ),
-                    self.evaluate(var0)[1],
-                )
-                self.assertAllCloseAccordingToType(
-                    np.array(
-                        [
-                            (
-                                1.0 * learning_rate
-                                - (1 - learning_rate) * lambda_ * 0.01 / norm1
-                            )
-                            * learning_rate
-                            - (1 - learning_rate) * lambda_ * 0.01 / norm1,
-                            (
-                                1.0 * learning_rate
-                                - (1 - learning_rate) * lambda_ * 0.01 / norm1
-                            )
-                            * learning_rate
-                            - (1 - learning_rate) * lambda_ * 0.01 / norm1,
-                        ]
-                    ),
-                    self.evaluate(var1)[2],
-                )
 
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_like_dist_belief_frobenius_cg01():
+    db_grad, db_out = _db_params_frobenius_cg01()
+    num_samples = len(db_grad)
+    var0 = tf.Variable([0.0] * num_samples)
+    grads0 = tf.constant([0.0] * num_samples)
+    ord = "fro"
+    cg_opt = cg_lib.ConditionalGradient(learning_rate=0.1, lambda_=0.1, ord=ord)
+
+    for i in range(num_samples):
+        grads0 = tf.constant(db_grad[i])
+        cg_opt.apply_gradients(zip([grads0], [var0]))
+        np.allclose(np.array(db_out[i]), var0.numpy())
```

Comment:
```suggestion
        np.testing.assert_allclose(np.array(db_out[i]), var0.numpy())
```

np.allclose returns a boolean.